sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/protocol/share_rest_client.hpp

```diff
@@ -1167,7 +1167,7 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
                     std::to_string(getPropertiesOptions.Timeout.GetValue())));
           }
           request.AddHeader(Details::HeaderVersion, getPropertiesOptions.ApiVersionParameter);
-          return GetPropertiesParseResult(context, pipeline.Send(context, request));
+          return GetPropertiesParseResult(context, pipeline.Send(request, context));
```

Comment:
Same here, and elsewhere. If you search for `"Result(context"`, you'll get ~50 results.

Reply:
@katmsft